runtime/experimental/export.Dockerfile

```diff
@@ -20,15 +20,17 @@ RUN rm -rf /opt/bref/data
 
 WORKDIR /opt
 COPY bootstraps/cli.bootstrap /tmp/cli.bootstrap
-RUN export PHP_ZIP_NAME=/export/php-$(php -r '$version = explode(".", phpversion());printf("%d.%d", $version[0], $version[1]);'); \
- mkdir -p /export; \
+RUN export PHP_ZIP_NAME=/export/php-$(php -r '$version = explode(".", phpversion());printf("%d.%d", $version[0], $version[1]);');
+RUN mkdir -p /export; \
  rm -rf /export/*; \
- rm -rf /bootstrap; \
- zip -qr ${PHP_ZIP_NAME}.zip . -x "*php-cgi"; \
- cp ${PHP_ZIP_NAME}.zip ${PHP_ZIP_NAME}-cli.zip; \
+ rm -rf /bootstrap;
+
+# Create the PHP CLI layer
+RUN zip --quiet --recurse-paths ${PHP_ZIP_NAME}-cli.zip . -x "*php-cgi"; \
  zip -d ${PHP_ZIP_NAME}-cli.zip bref/sbin/php-fpm bin/php-fpm; \
  cp /tmp/cli.bootstrap /bootstrap; \
  chmod 755 /bootstrap; \
- zip -u ${PHP_ZIP_NAME}-cli.zip /bootstrap; \
- cp ${PHP_ZIP_NAME}.zip ${PHP_ZIP_NAME}.fpm.zip; \
- zip -d ${PHP_ZIP_NAME}.fpm.zip bref/bin/php /bin/php
+ zip -u ${PHP_ZIP_NAME}-cli.zip /bootstrap;
+
+# Create the PHP FPM layer
+RUN zip --quiet --recurse-paths ${PHP_ZIP_NAME}-fpm.zip . -x "*php-cgi"
```

Comment:
I separated the two just for the sake of readability.

Also I fixed `php-7.2.fpm` into `php-7.2-fpm`.

Note that the FPM runtimes also contains the `php` CLI binary. I originally did that on purpose thinking that there may be scenarios where one would want to run CLI script from a PHP-FPM process. Now that I'm writing this paragraph I'm starting to doubt that 🤔 That wouldn't make sense on lambda.

I'm not considering remove the binary… Thoughts?

Reply:
Just removed it, we can add it again if we see a good use case pop up.

Reply:
Aye. There might be a use case for it, but I can't think of it. I believe that in a perfect world, if http needs to run cli, it would do it by calling a different lambda job. That could either be synchronously in a blocking call or asynchronously and send it to a queue.